CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added required `array` argument when invoking `SerializationError` ([#419](https://github.com/opensearch-project/opensearch-js/pull/419))
 - Removed unnecessary `data` argument when invoking `OpenSearchClientError` ([#421](https://github.com/opensearch-project/opensearch-js/pull/421))
 - Fixed typos in `ConnectionPool` ([#427](https://github.com/opensearch-project/opensearch-js/pull/427))
+- Added `BaseConnectionPool` `markDead` and `markAlive` methods implementation ([#430](https://github.com/opensearch-project/opensearch-js/pull/430))
```

Comment:
Try to write this from the user's perspective, before they tried to call `markDead` and it didn't work. You're fixing it, so let's say "Fixed `BaseConnectionPool#markDead` and `#markAlive`"